internal/credential/vault/ssh_certificate_credential_library.go

```diff
@@ -0,0 +1,98 @@
+package vault
+
+import (
+	"github.com/hashicorp/boundary/internal/credential"
+	"github.com/hashicorp/boundary/internal/credential/vault/store"
+	"github.com/hashicorp/boundary/internal/oplog"
+	"google.golang.org/protobuf/proto"
+)
+
+const (
+	KeyTypeEcdsa   = "ecdsa"
+	KeyTypeEd25519 = "ed25519"
+	KeyTypeRsa     = "rsa"
+)
+
+// A CredentialLibrary contains a Vault path and is owned by a credential
+// store.
```

Comment:
This comment needs updating

```suggestion
// SSHCertificateCredentialLibrary is a credential library that issues
// ssh certificate using the vault ssh secret engine.
// See: https://developer.hashicorp.com/vault/api-docs/secret/ssh#sign-ssh-key
```